test/cpp/test_symint.cpp

```diff
@@ -149,21 +149,37 @@ TEST(SymintTest, TestDynamicSymintArithmetic) {
 
   c10::SymInt a =
       c10::make_intrusive<XLASymIntNodeImpl>(size_abs_node)->toSymInt();
-
   c10::SymInt b =
       c10::make_intrusive<XLASymIntNodeImpl>(size_relu_node)->toSymInt();
 
-  c10::SymInt c = a * b;
+  c10::SymInt c = a + b;
```

Comment:
nit: add a short comment to separate the three tests

Reply:
done